### PR TITLE
CANS-643: change assessment status from SUBMITTED to COMPLETED

### PR DESCRIPTION
**Pull request by LeonidMarushevskyi**
## JIRA 
https://osi-cwds.atlassian.net/browse/CANS-643
## Description

Change assessment columns name due to change of status from SUBMITTED to COMPLETED.

## Tests
- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [x] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters and static analyzers for the project (SonarQube, eslint, rubocop, etc).
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: src/main/java/gov/ca/cwds/cans/domain/entity/Assessment.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
 /** An Assessment. */
 @Entity
 @Table(name = "assessment")
-@NamedQuery(name = NQ_ALL, query = "FROM Assessment a order by status asc, event_date desc")
+@NamedQuery(name = NQ_ALL, query = "FROM Assessment a order by status desc, event_date desc")
```
Comment (denys-davydov) [inline src/main/java/gov/ca/cwds/cans/domain/entity/Assessment.java]: Nice catch here

File: src/test/java/gov/ca/cwds/cans/rest/resource/AssessmentResourceTest.java [hunk 2]
```diff
@@ -206,6 +206,8 @@ public void searchAssessments_findsFourSortedRecords() throws IOException {
 
     // then
     assertThat(actualResults.length, is(4));
+    System.out.println("!!! assessmentIds" + assessmentIds);
```
Comment (denys-davydov) [inline src/test/java/gov/ca/cwds/cans/rest/resource/AssessmentResourceTest.java]: Seems to be forgotten to-delete lines here

File: src/main/java/gov/ca/cwds/cans/domain/enumeration/AssessmentStatus.java [hunk 3]
```diff
@@ -3,6 +3,6 @@
 /** The AssessmentStatus enumeration. */
 public enum AssessmentStatus {
   IN_PROGRESS,
-  SUBMITTED,
+  COMPLETED,
   APPROVED
```
Comment (denys-davydov) [inline src/main/java/gov/ca/cwds/cans/domain/enumeration/AssessmentStatus.java]: Let's delete APPROVED status if it is not used

File: src/test/java/gov/ca/cwds/cans/rest/resource/AssessmentResourceTest.java (hunk 2 above)
Comment (LeonidMarushevskyi) [reply src/test/java/gov/ca/cwds/cans/rest/resource/AssessmentResourceTest.java]: Yes, you are right. Removing log.

**Comment by LeonidMarushevskyi** [comment]
Comments were addressed.

**Pull request merged**